package.json

```diff
@@ -1,32 +1,18 @@
 {
-  "name": "embrace-io",
-  "version": "3.51.0",
-  "private": "false",
-  "author": {
-    "name": "Embrace.io",
-    "email": "truenorth@embrace.io"
-  },
-  "homepage": "https://embrace.io/",
-  "description": "A React Native wrapper for Embrace SDK",
-  "main": "dist/src/embrace.js",
-  "types": "dist/src/embrace.d.ts",
-  "files": [
-    "/dist",
-    "/android",
-    "/ios",
-    "RNEmbrace.podspec",
-    "react-native.config.js"
+  "name": "embrace-react-native-monorepo",
+  "private": true,
+  "description": "Monorepo for packages that power Embrace's React Native SDK.",
+  "workspaces": [
+    "packages/*"
   ],
+  "packageManager": "yarn@4.3.1",
+  "engines": {
+    "node": ">=18.0.0"
+  },
   "scripts": {
```

Comment:
@TheMutsi I was hoping to get rid of old scripts here, are there any I removed that are still being used? Some additional questions:
- I removed the `-tag-version-prefix='rn-os-modules-'` part of the publish command, is it needed?
- should "build" be changed to "npx lerna run tsc" or "npx lerna run build"? I see that it generated a `/dist` folder at the root level but I assumed we were publishing the ones in packages/*/dist. The CI workflow also does a `yarn build` after running lint and test that I wasn't sure if it was needed
- I moved 'prettier' to be part of 'lint' as it didn't seem like that was being run as part of the CI previously

Reply:
`-tag-version-prefix='rn-os-modules-` was needed because it was giving errors with tags when we deploy the NPM core and modules.
You can remove yarn build, the last change that I made in the repo was moving the principal SDK to a lerna package called core. Now I see there are some scripts that I forgot to delete. The most important is publish-modules now, then test/jest, linters etc

Reply:
got it, okay I removed the rest and just kept `publish`, `lint, `test`. I removed `tag-version` for now and we can see what errors we get when trying to deploy